### PR TITLE
Fixed vHost bug and added usage example

### PR DESCRIPTION
**Pull request by KevinMellott91**

### CODE REVIEW

**Comment by nelsou** [comment]
+1

File: src/main/scala/com/stratio/example/RabbitMQConsumer.scala [hunk 1]
```diff
@@ -0,0 +1,64 @@
+/**
```
Comment (danielcsant) [inline src/main/scala/com/stratio/example/RabbitMQConsumer.scala]: Please, move this file to src/**test**/scala/com/stratio/example/RabbitMQConsumer.scala. We don't want it to be packaged

Comment (KevinMellott91) [inline src/main/scala/com/stratio/example/RabbitMQConsumer.scala]: Ah good point; making the update now.

**Comment by danielcsant** [comment]
Very helpfull! Thanks for your contribution

**Pull request merged**